src/Controller/AboutController.php

```diff
@@ -222,10 +222,23 @@ public function earlyCareerAction(Request $request) : Response
                     new Paragraph('For more information, take a look at this <a href="'.$this->get('router')->generate('inside-elife-article', ['id' => '140901c3']).'">video message from the ECAG</a>.')
                 ),
                 'Community Ambassadors' => 'We convene and facilitate a worldwide community of like-minded researchers, led by the ECAG. The eLife Community Ambassadors champion responsible behaviours in science among colleagues and create and deliver solutions that accelerate positive changes in scholarly culture.',
-                'Involvement in peer review' => 'eLife encourages reviewers to involve junior colleagues as co-reviewers; we involve outstanding early-stage researchers as reviewers <a href="'.$this->get('router')->generate('inside-elife-article', ['id' => '31a5173b']).'">in their own right</a>; and we enable all reviewers to receive credit for their contributions through services such as Publons and ORCID',
+                'Involvement in peer review' => $this->render(
+                    new Paragraph('eLife encourages editors to nominate and involve early-career researchers in the review process. The <a href="'.$this->get('router')->generate('inside-elife', ['id' => '31a5173b']).'">eLife pool of early-career reviewers</a> aims to provide outstanding early-stage researchers the opportunity to peer review manuscripts. Members of the pool are nominated and/or approved by the eLife editors.'),
```

Comment:
`generate('inside-elife'...` links to the inside eLife page. Replaced code had it as `generate('inside-elife-article'...` that linked to a specific article...